set-up.R

```diff
@@ -53,33 +53,17 @@ remove_style = function(x){
 
 # function to add model_output_table class to all tables
 add_table_class <- function(x){
-  
   x <- gsub("<table>", "<table class='model_output_table'>", x)
   x
 }
 
-save_formats <- function(to_save, name = F, csv = F){
+save_formats <- function(to_save, name = F){
   if (name == F){
     name <- substitute(to_save)
   }
   saveRDS(to_save, file.path(pct_data, region, paste0(name, ".Rds")))
-  
-  # Simplify data checked with before and after using:
-  # plot(l$gendereq_sideath_webtag)
-  to_save@data <- round_df(to_save@data, 5)
-  
-  # Simplify geom
-  geojson_write( ms_simplify(to_save, keep = 0.1), file = file.path(pct_data, region, name))
```

Comment:
OK - we can always re-generate them after the event. From the perspective of making data available, however, I think it's really useful to have the geojson in the data repo, however. Imagine you're a cycling campaigner who uses QGIS/Python but has no idea about R. Having a .geojson available, if our server goes down, would be beneficial for you, no?